STYLEGUIDE.md

```diff
@@ -1,8 +1,14 @@
 # Style Guide
 
-This is WIP! It is definitely not complete. We just starting putting this together.
+This is a general style guide that shall govern over the Flame Engine repository. The aim is to keep a common stable general environment.
```

Comment:
Max 100 in line length

Reply:
Oh, this should also be mentioned in this doc I guess

Reply:
done